src/functions/helper_parser.ts

```diff
@@ -0,0 +1,282 @@
+  "in^2": { ...transformFromFactor(Math.pow(IN2M, 2)), category: "area", order: 2 },
+  "ft^2": { ...transformFromFactor(Math.pow(FT2M, 2)), category: "area", order: 2 },
+  "yd^2": { ...transformFromFactor(Math.pow(YD2M, 2)), category: "area", order: 2 },
+  ar: { ...transformFromFactor(100), category: "area" },
+  Morgen: { ...transformFromFactor(2500), category: "area" },
+  uk_acre: { ...transformFromFactor(4046.8564224), category: "area" },
+  us_acre: { ...transformFromFactor(4046.8726098743), category: "area" },
+  ha: { ...transformFromFactor(1e4), category: "area" },
+  "mi^2": { ...transformFromFactor(Math.pow(MI2M, 2)), category: "area", order: 2 },
+  "Nmi^2": { ...transformFromFactor(Math.pow(NMI2M, 2)), category: "area", order: 2 },
+  "ly^2": { ...transformFromFactor(Math.pow(LY2M, 2)), category: "area", order: 2 },
+
+  // INFORMATION UNITS : Standard = bit
+  bit: { ...standard, category: "information" },
+  byte: { ...transformFromFactor(8), category: "information" },
+
+  // SPEED UNITS : Standard = m/s
+  "m/s": { ...standard, category: "speed" },
+  "m/hr": { ...transformFromFactor(1 / 3600), category: "speed" },
+  "km/hr": { ...transformFromFactor(1 / 3.6), category: "speed" },
+  mph: { ...transformFromFactor(0.44704), category: "speed" },
+  kn: { ...transformFromFactor(0.5144444444), category: "speed" },
+  admkn: { ...transformFromFactor(0.5147733333), category: "speed" },
+};
+
+export const UNITS_ALIASES = {
+  shweight: "cwt",
+  lcwt: "uk_cwt",
+  hweight: "uk_cwt",
+  LTON: "uk_ton",
+  brton: "uk_ton",
+  pc: "parsec",
+  Pica: "Picapt",
+  d: "day",
+  mn: "min",
+  s: "sec",
+  p: "Pa",
+  at: "atm",
+  dy: "dyn",
+  ev: "eV",
+  hh: "HPh",
+  wh: "Wh",
+  btu: "BTU",
+  h: "HP",
+  cel: "C",
+  fah: "F",
+  kel: "K",
+  us_pt: "pt",
+  L: "l",
+  lt: "l",
+  ang3: "ang^3",
+  ft3: "ft^3",
+  in3: "in^3",
+  ly3: "ly^3",
+  m3: "m^3",
+  mi3: "mi^3",
+  yd3: "yd^3",
+  Nmi3: "Nmi^3",
+  Picapt3: "Picapt^3",
+  "Pica^3": "Picapt^3",
+  Pica3: "Picapt^3",
+  regton: "GRT",
+  ang2: "ang^2",
+  ft2: "ft^2",
+  in2: "in^2",
+  ly2: "ly^2",
+  m2: "m^2",
+  mi2: "mi^2",
+  Nmi2: "Nmi^2",
+  Picapt2: "Picapt^2",
+  "Pica^2": "Picapt^2",
+  Pica2: "Picapt^2",
+  yd2: "yd^2",
+  "m/h": "m/hr",
+  "m/sec": "m/s",
+};
+
+export const UNIT_PREFIXES = {
+  "": 1,
+  Y: 1e24,
+  Z: 1e21,
+  E: 1e18,
+  P: 1e15,
+  T: 1e12,
+  G: 1e9,
+  M: 1e6,
+  k: 1e3,
+  h: 1e2,
+  da: 1e1,
+  e: 1e1,
+  d: 1e-1,
+  c: 1e-2,
+  m: 1e-3,
+  u: 1e-6,
+  n: 1e-9,
+  p: 1e-12,
+  f: 1e-15,
+  a: 1e-18,
+  z: 1e-21,
+  y: 1e-21,
+  Yi: Math.pow(2, 80),
+  Zi: Math.pow(2, 70),
+  Ei: Math.pow(2, 60),
+  Pi: Math.pow(2, 50),
+  Ti: Math.pow(2, 40),
+  Gi: Math.pow(2, 30),
+  Mi: Math.pow(2, 20),
+  ki: Math.pow(2, 10),
+};
+
+const TRANSLATED_CATEGORIES = Object.assign(
+  {},
+  ...Array.from(new Set(Object.values(UNITS).map((u) => u.category)), (u) => ({ [u]: _t(u) }))
```

Comment:
still doesn't work. There can never be variables inside `_t(...)`, only hard-coded static strings.
We need an explicit and hard-coded mapping

```js
const TRANSLATED_CATEGORIES = {
    volume: _t("Volume"),
    ...
}
```